man/rhsmcertd.8

```diff
@@ -62,10 +62,10 @@ Resets the interval for checking for and replacing expired subscriptions. This v
 file are used (unless the argument is passed again).
 
 .TP
-.B -m, --max-splay-minutes=MINUTES
-Sets the maximum time to offset the checks done by this tool. This value is in minutes. The default value is 10. It may be useful in large deployments to increase this value to reduce peak load on the entitlement server these systems are registered to. Setting this value to 0 effectively turns off this feature. This interval is in effect until the daemon restarts, and then the values in the
+.B -s, --no-splay
+Disables splay. This value is in effect until the daemon is restarted. Unless passed in again, the value of "splay" from the
```

Comment:
> Unless passed in again

What does this mean? Does this mean `-s` is a toggle? or that running `rhsmcertd` again without `-s` will reset to default? Or that if `-s` is not present, then behavior is dependent on the config file? It's not particularly clear.